platformio.ini

```diff
@@ -128,28 +128,28 @@ board_build.extra_flags = -DBOARD_HAS_PSRAM '-DEMSESP_DEFAULT_BOARD_PROFILE="S32
 ;
 
 [env:s_4M]
-extends = espressi32_base_tasmota
+extends = espressif32_base_tasmota
 board = esp32dev
 board_upload.flash_size = 4MB
 board_build.partitions = esp32_partition_4M.csv
 board_build.extra_flags = '-DEMSESP_DEFAULT_BOARD_PROFILE="S32"'
```

Comment:
platformIO is a beast, putting the boardprofile here instead of build_flags disables the platform menu in vscode. Can not erase a chip.

Reply:
I still see the erase option in the menu

![alt text](image.png)

Reply:
Restarting vscode brings my Platform folder back, Don't now why it was not shown after first loading your changes.